app/main.py

```diff
@@ -1,8 +1,67 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: List) -> float:
+        total_income = 0.0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income = self.wash_single_car(car)
+                total_income += income
+        return round(total_income, 1)
+
+    def calculate_washing_price(self, car: Car) -> float:
+        if not isinstance(car, Car):
+            raise ValueError("The argument must be a Car instance")
+        if car.clean_mark >= self.clean_power:
+            return 0.0
+        washing_price = (
```

Comment:
You don't need to create a variable here since you not going to reuse it anywhere